Angular/my-app5/src/forms/forms.component.ts

```diff
@@ -0,0 +1,64 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, FormsModule, ReactiveFormsModule, Validators } from '@angular/forms';
+
+
+interface address{
```

Comment:
better to create a seperate file. Interface names should startwith I, pascalcasing

Reply:
Interface given in seperate folder and given I pascalcasing for interface.